README.md

```diff
@@ -29,54 +61,96 @@ There are some configuration options for the sysadmins out there. All configurat
 * `path` *(string)* Path to the application data folder, which contains the private key, message attachment data (blobs) and the leveldb backend. Defaults to `$HOME/.ssb`.
 * `master` *(array)* Pubkeys of users who, if they connect to the Scuttlebot instance, are allowed to command the primary user with full rights. Useful for remotely operating a pub. Defaults to `[]`.
 * `logging.level` *(string)* How verbose should the logging be. Possible values are error, warning, notice, and info. Defaults to `notice`.
+* `remote` ... TODO (this was formerly undocumented but came up in conversation)
 
-### `connections`
+Deprecated Options:
```

Comment:
Great idea! Probably a good idea to warn about that if they are overwritten.

Reply:
do you mean that the raw values are being removed from the final config? I've added a note about that now

Reply:
Given what dinosaur said about env variables scratch my comment.